src/types/index.ts

```diff
@@ -15,5 +15,40 @@ export interface StoreState {
 
 // TODO: Complete Piece Interface.
 export interface Piece {
-
-}
+  pieceElementId: string;
+  initialState: InitialState;
+  deckPieceIndex: number;
+  }
+  
+  export interface Pieces {
+  [pieceIndex: string]: Piece;
+  }
+  
+  interface InitialState {
+  x: number;
+  y: number;
+  zDepth: number;
+  currentImageIndex: number;
+  cardVisibility: CardVisibility;
+  rotationDegrees: number;
+  drawing: Drawing;
+  }
+  
+  interface Line {
+  userId: string;
+  timestamp: number/*firebase.database.ServerValue.TIMESTAMP*/;
+  color: string;
+  lineThickness: number;
+  fromX: number;
+  fromY: number;
+  toX: number;
+  toY: number;
+  }
+  
+  interface Drawing {
+  [lineId: string]: Line;
+  }
+  
+  interface CardVisibility {
+  [participantIndex: string]: boolean;
+  }
```

Comment:
Add a new line in the end of the file.
Preferably install editorconfig extension in your favorite text editor.